tests/integration_tests/modules/test_lxd.py

```diff
@@ -45,3 +52,28 @@ def test_bridge(self, class_client):
         raw_network_config = class_client.execute("lxc network show lxdbr0")
         network_config = yaml.safe_load(raw_network_config)
         assert "10.100.100.1/24" == network_config["config"]["ipv4.address"]
+
+
+def validate_storage(validate_client):
+    verify_clean_log(
+        validate_client.read_from_file("/var/log/cloud-init.log"),
+        ignore_deprecations=False,
+    )
```

Comment:
can we also assert that the right package got installed by cc_lxd.
```suggestion
def validate_storage(validate_client, pkg_name):
    log = validate_client.read_from_file("/var/log/cloud-init.log")
    verify_clean_log(log,  ignore_deprecations=False)
    assert f"install', '{pkg_name}' in log
```

or maybe an re.match(r"apt.*install.*{pkg_name}", log) is not None